app/services/samples/transfer_service.rb

```diff
@@ -62,18 +63,39 @@ def transfer(new_project_id, sample_ids) # rubocop:disable Metrics/MethodLength,
         end
         next
       end
+
       unless not_found_sample_ids.empty?
         @project.errors.add(:samples,
                             I18n.t('services.samples.transfer.samples_not_found',
                                    sample_ids: not_found_sample_ids.join(', ')))
       end
 
       if transferred_samples_ids.count.positive?
+        create_activities
+
         @project.namespace.update_metadata_summary_by_sample_transfer(transferred_samples_ids,
                                                                       new_project_id)
       end
 
       transferred_samples_ids
     end
+
+    def create_activities
```

Comment:
**nitpick (blocking):** Would it be possible to add the `sample_ids` or `sample_puids` to the activities. That way we know exactly what was transferred and then we could update the view to show the samples transferred.

Reply:
As discussed we will add an enhancement in a future PR to display the sample puids but for now its been updated to display `X number of samples were deleted`, `X number of samples were cloned`, and `X number of samples were transferred` in 611559cd2af78fbea14c8b415e503768682809f5. The activities themselves have the sample ids and puids stored in them so only the front end should need to be updated in that pr